hedera-node/src/test/java/com/hedera/services/txns/consensus/SubmitMessageTransitionLogicTest.java

```diff
@@ -92,6 +94,7 @@ public void rubberstampsSyntax() {
 	public void hasCorrectApplicability() {
 		// given:
 		givenValidTransactionContext();
+		given(globalDynamicProperties.messageMaxBytesAllowed()).willReturn(1024);
```

Comment:
Those `given(globalDynamicProperties.messageMaxBytesAllowed()).willReturn(1024);` should be in the `setup()` instead of being duplicated across all tests.

Reply:
Yeah. Moved most of this statement to setup()